packages/forma-36-react-components/src/components/EntityList/EntityListItem/EntityListItem.tsx

```diff
@@ -0,0 +1,246 @@
+   */
+  entityType?: 'entry' | 'asset';
+  /**
+   * Loading state for the component - when true will display loading feedback to the user
+   */
+  isLoading?: boolean;
+  /**
+   * The action to be performed on click of the EntryCard
+   */
+  onClick?: MouseEventHandler;
+  /**
+   * The href for the component. Will render the card as an `a` element for native browser link handling
+   */
+  href?: string;
+  /**
+   * Class names to be appended to the className prop of the component
+   */
+  className?: string;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id)
+   */
+  testId?: string;
+} & typeof defaultProps;
+
+const defaultProps = {
+  testId: 'cf-ui-entity-list-item',
+  entityType: 'entry',
+};
+
+export class EntityListItem extends Component<EntityListItemProps> {
+  static defaultProps = defaultProps;
+
+  renderIcon() {
+    const icon = this.props.entityType === 'asset' ? 'Asset' : 'Entry';
+
+    return <Icon icon={icon} color="muted" />;
+  }
+
+  renderThumbnail() {
+    return (
+      <img
+        src={this.props.thumbnailUrl}
+        className={styles['EntityListItem__thumbnail']}
+        alt={this.props.thumbnailAltText}
+      />
+    );
+  }
+
+  renderStatus = (status: EntityListItemStatus) => {
+    let label: string;
+    let type: TagType;
+
+    switch (status) {
+      case 'archived':
+        label = 'archived';
+        type = 'negative';
+        break;
+
+      case 'changed':
+        label = 'changed';
+        type = 'primary';
+        break;
+
+      case 'published':
+        label = 'published';
+        type = 'positive';
+        break;
+
+      default:
+        label = 'draft';
+        type = 'warning';
+    }
+
+    return (
+      <div className={styles['EntityListItem__status']}>
+        <Tag tagType={type}>{label}</Tag>
+      </div>
+    );
+  };
+
+  renderLoadingCard() {
+    return (
+      <article className={styles['EntityListItem__inner']}>
+        <SkeletonContainer clipId="f36-entity-list-item-skeleton">
+          <SkeletonImage height={46} width={46} />
+          <SkeletonBodyText numberOfLines={2} lineHeight={18} offsetLeft={54} />
+        </SkeletonContainer>
+      </article>
+    );
+  }
+
+  render() {
+    const {
+      className,
+      testId,
+      title,
+      description,
+      contentType,
+      entityType,
+      thumbnailUrl,
+      thumbnailAltText,
+      status,
+      dropdownListElements,
+      withDragHandle,
+      isDragActive,
+      cardDragHandleProps,
+      isLoading,
+      onClick,
+      href,
+      ...otherProps
+    } = this.props;
+
+    const classNames = cn(styles.EntityListItem, className, {
+      [styles['EntityListItem--drag-active']]: isDragActive,
+      [styles['EntityListItem--is-interactive']]: onClick || href,
+    });
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const Element = onClick ? 'a' : 'article';
```

Comment:
why is it complaining is this not just Element: string ?

Reply:
Good spot - this wasn't actually needed 😅 - removed!